MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Tabs.xaml

```diff
@@ -0,0 +1,414 @@
+                                              ContentSource="SelectedContent"
+                                              ContentStringFormat="{TemplateBinding ContentStringFormat}"
+                                              ContentTemplate="{TemplateBinding ContentTemplate}"
+                                              ContentTemplateSelector="{TemplateBinding ContentTemplateSelector}">
+                            </ContentPresenter>
+                        </wpf:ColorZone>
+                    </DockPanel>
+                    <ControlTemplate.Triggers>
+                        <Trigger Property="wpf:ColorZoneAssist.Mode" Value="Standard">
+                            <Setter TargetName="PART_ColorZone" Property="Background" Value="Transparent"/>
+                        </Trigger>
+                        <Trigger Property="TabStripPlacement" Value="Top">
+                            <Setter TargetName="tabgrid" Property="DockPanel.Dock" Value="Top"/>
+                            <Setter Property="wpf:ShadowAssist.ShadowEdges" Value="Bottom"/>
+                            <Setter Property="BorderThickness" Value="0,0,0,1" />
+                        </Trigger>
+                        <Trigger Property="TabStripPlacement" Value="Bottom">
+                            <Setter TargetName="tabgrid" Property="DockPanel.Dock" Value="Bottom"/>
+                            <Setter Property="wpf:ShadowAssist.ShadowEdges" Value="Top"/>
+                            <Setter Property="BorderThickness" Value="0,1,0,0" />
+                        </Trigger>
+                        <Trigger Property="TabStripPlacement" Value="Left">
+                            <Setter TargetName="tabgrid" Property="DockPanel.Dock" Value="Left"/>
+                            <Setter TargetName="HeaderPanel" Property="Rows" Value="0"/>
+                            <Setter TargetName="HeaderPanel" Property="Columns" Value="1"/>
+                            <Setter Property="wpf:ShadowAssist.ShadowEdges" Value="Right"/>
+                            <Setter Property="BorderThickness" Value="0,0,1,0"/>
+                        </Trigger>
+                        <Trigger Property="TabStripPlacement" Value="Right">
+                            <Setter TargetName="tabgrid" Property="DockPanel.Dock" Value="Right"/>
+                            <Setter TargetName="HeaderPanel" Property="Rows" Value="0"/>
+                            <Setter TargetName="HeaderPanel" Property="Columns" Value="1"/>
+                            <Setter Property="wpf:ShadowAssist.ShadowEdges" Value="Left"/>
+                            <Setter Property="BorderThickness" Value="1,0,0,0"/>
+                        </Trigger>
+                    </ControlTemplate.Triggers>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+    </Style>
+
+
+    <Style TargetType="{x:Type TabItem}" x:Key="MaterialDesignNavigationRailTabItem">
```

Comment:
Should this be getting set above so that the tabs don't all have to set it explicitly.
```
<Setter Property="ItemContainerStyle" Value="{StaticResource MaterialDesignNavigationRailTabItem}" />
```

Reply:
I'm getting a runtime exception by doing this:

System.Windows.Markup.XamlParseException: '' A defined property 'System.Windows.FrameworkElement.Style' raises an exception. 'Line number' 30 'and line position' 22 '. '

InvalidCastException: It is not possible to convert an object of type 'MS.Internal.NamedObject' to type 'System.Windows.Style'.

Reply:
I've found a solution!! Moving the TabItem' Style above TabControl' Style it works!

![image](https://user-images.githubusercontent.com/15158923/92908884-038bf200-f3fd-11ea-9f4d-b4b2f8c17e37.png)